libs/parser-sql/src/postgresParser.ts

```
@@ -0,0 +1,1652 @@
+    // elements
+    parameterRule: () => ParameterAst
+    identifierRule: () => IdentifierAst
+    stringRule: () => StringAst
+    integerRule: () => IntegerAst
+    decimalRule: () => DecimalAst
+    booleanRule: () => BooleanAst
+    nullRule: () => NullAst
+
+    constructor(tokens: TokenType[], recovery: boolean) {
+        super(tokens, {recoveryEnabled: recovery})
+        const $ = this
```

Biome

[error] This aliasing of this is unnecessary.

Arrow functions inherits `this` from their enclosing scope.
Safe fix: Use this instead of an alias.


(lint/complexity/noUselessThisAlias)